storage/acl/addBucketACL.go

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package acl
+
+// [START bucket_add_acl]
```

Comment:
Would you mind switching to the canonical region tags while you're updating these?

Reply:
I believe these are the ACL samples (I haven't cross referenced with the samples here):

* storage_add_bucket_owner
* storage_remove_bucket_owner
* storage_add_bucket_default_owner
* storage_remove_bucket_default_owner
* storage_print_bucket_acl
* storage_print_bucket_acl_for_user
* storage_add_file_owner
* storage_remove_file_owner
* storage_print_file_acl
* storage_print_file_acl_for_user

cc @crwilcox

Reply:
Yes, I see these samples in Python, they are similar to Go's (small diffs, like in Python we're setting const role, while in Go we're setting role passed as arg). I'll rename Go samples tags for ACL, it's just a few minutes after all, but I'm not sure if these samples are actually used - I didn't find them in docs